core/account-id/src/lib.rs

```diff
@@ -132,6 +150,29 @@ impl AccountId {
             .map_or(false, |s| !s.contains('.'))
     }
 
+    /// Returns `true` if the `AccountId` is a 40 characters long hexadecimal prefixed with '0x'.
```

Comment:
nit: It's slighly unclear if the total length is 40 or 42 characters, can you clarify that please?